WSLDVCPlugin/WSLDVCCallback.cpp

```diff
@@ -510,21 +512,27 @@ class WSLDVCCallback :
             }
         }
 
-        if (updateAppList.appGroupLength && (wcsstr(updateAppList.appGroup, L"..") != NULL))
+        if (updateAppList.appGroupLength)
         {
-            DebugPrint(L"group name can't contain \"..\" %s\n", updateAppList.appGroup);
-            return E_FAIL;
+            if ((wcsstr(updateAppList.appGroup, L"..") != NULL) ||
+                (wcsstr(updateAppList.appGroup, L"\"") != NULL))
+            {
+                DebugPrint(L"group name can't contain '..' or '\".', %s\n", updateAppList.appGroup);

```

Comment:
Small nit, but is this period supposed to be here?

Reply:
Thanks, removed.